### DISCUSSION TITLE
Preserve original filename in metadata

### DISCUSSION THREAD

**Discussion by juahan** [jonaswinkler/paperless-ng#1576] [Feature Requests]
Currently the original filename is the title, but my workflow includes rewriting the title. I might still have important information in the original filename so I would like to keep it as well in the metadata. Could be done automatically, but also just a custom field would do.

**Comment by moorgrove** [comment]
+1
Would be a great function when renaming the title and later want to double check that a skipped file (doublet) really exist in the library. Right now the original filename is not searchable?

**Comment by tompsg-git** [comment]
> Currently the original filename is the title, but my workflow includes rewriting the title. 

How did you do that?
